packages/components/badge/src/Badge.styled.tsx

```diff
@@ -1,19 +1,10 @@
 import tokens from '@contentful/f36-tokens';
 import styled from '@emotion/styled';
 import { css } from '@emotion/core';
-import type { EntityStatus, TagType } from './types';
+import type { BadgeVariant } from './types';
 
-export interface TagProps {
-  tagType?: TagType;
-  style?: React.CSSProperties;
-  className?: string;
-  testId?: string;
-  children: React.ReactNode;
-  entityStatusType?: EntityStatus;
-}
-
-const tagTypeToStyles = ({ tagType }: { tagType: TagType }) => {
-  switch (tagType) {
+const tagTypeToStyles = ({ variant }: { variant: BadgeVariant }) => {
```

Comment:
Should the function be renamed with variant?

Reply:
renamed